CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 # the terms of the Apache License 2.0 which accompanies this distribution.     #
 # ============================================================================ #
 
-cmake_minimum_required(VERSION 3.22 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.26 FATAL_ERROR)
```

Comment:
Is this necessary? The LLVM we're using only requires 3.13 and suggests 3.20.

Reply:
I can move this to a separate PR. 3.26 is necessary for the Python wheels. If specify a lower version here, then we need to make sure that we use the lower cmake version in the dev dependencies used to build the docker file, so that building with the lower cmake version is tested.